.github/workflows/publish-storybook.yml

```
@@ -1,6 +1,9 @@
 name: Publish Storybook to Chromatic
 
-on: push
+on:
+  push:
+    branches-ignore:
+      - 'refs/**'
```

To make this more specific and clarify the intent, maybe...
```suggestion
      - 'refs/tags/*'
```

---

I was thinking that we would want to ignore anything that started with `refs/`, as those aren't real branches.

---

I'm a little skeptical that this might end up filtering out everything, since branches are refs too (`refs/heads/...` instead of `refs/tags/...`).

https://git-scm.com/book/en/v2/Git-Internals-Git-References

But the docs say that these patterns match against `refs/heads`.

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-excluding-branches

---

Oooh, it's running on pushes of either branches or tags.

This StackOverflow answer says to avoid running on pushes to tags, we could specify `branches: ['**']` instead of `branches-ignore`.
https://stackoverflow.com/a/70743841

---

@nawatts I changed the syntax, having read further to https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#example-excluding-branches-and-tags